### PR TITLE
Add KBO-5000 keyboard

### PR DESCRIPTION
**Pull request by nooges**
## Description

Adds KBO-5000 keyboard

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/keebio/kbo5000/rev1/rev1.c [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+void eeconfig_init_kb(void) {
+#ifdef BACKLIGHT_ENABLE
+    backlight_enable();
+    backlight_level(3);
+#endif
+#ifdef RGBLIGHT_ENABLE
+    rgblight_enable(); // Enable RGB by default
+    rgblight_sethsv(0, 255, 128);  // Set default HSV - red hue, full saturation, medium brightness
+    rgblight_sethsv(0, 255, 255);  // Set default HSV - red hue, full saturation, full brightness
```
Comment (drashna) [inline keyboards/keebio/kbo5000/rev1/rev1.c:25]: Intentional?

Comment (nooges) [reply keyboards/keebio/kbo5000/rev1/rev1.c]: Fixed now

File: keyboards/keebio/kbo5000/rev1/rev1.h [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+#include "kbo5000.h"
+#include "quantum.h"
+
+#ifdef USE_I2C
+#include <stddef.h>
+#ifdef __AVR__
+    #include <avr/io.h>
+    #include <avr/interrupt.h>
+#endif
+#endif
```
Comment (drashna) [inline keyboards/keebio/kbo5000/rev1/rev1.h:12]: Is this actually needed?

Comment (fauxpark) [reply keyboards/keebio/kbo5000/rev1/rev1.h:12]: Probably not.

Comment (nooges) [reply keyboards/keebio/kbo5000/rev1/rev1.h]: Removed it

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**